### PR TITLE
Refactor EVP- LRQA-83371 | Master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LRQA-83371](https://liferay.atlassian.net/browse/LRQA-83371)
Some tests are failing due to this [bug](https://liferay.atlassian.net/browse/LPS-200541)

### CODE REVIEW

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/JSONEVPAPI.macro [hunk 1]
```diff
@@ -48,6 +48,8 @@ definition {
 
 		var organizationId = JSONCurlUtil.post(${curl}, "$.id");
 
+		static var staticorganizationId = ${organizationId};
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/JSONEVPAPI.macro:51]: Fix to staticOrganizationId

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase [hunk 2]
```diff
@@ -173,7 +185,7 @@ definition {
 			EVPRequest.goToCreateRequest();
 		}
 
-		task ("Then the 'phone number' field is enabled and He fill this") {
+		task ("Then the 'phone number' field is enabled and is possible fill this") {
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase:188]: Fix to "Then the 'phone number' field is enabled and it is possible to fill it"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase [hunk 3]
```diff
@@ -203,61 +215,35 @@ definition {
 			EVPRequest.goToCreateRequest();
 		}
 
-		task ("Then the 'phone number' field is disabled and it's not possible fill") {
+		task ("Then the 'phone number' field is disabled and it's not possible fill this") {
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase:218]: Fix to "Then the 'phone number' field is disabled and it's not possible to fill it"

**Comment by MGabriellaS** [comment]
ci:test:sf

**Review by vinicius-paes — APPROVED** [review]

**Comment by vinicius-paes** [comment]
ci:forward

**Pull request closed without merging**